src/hal/cortex_m0/isr.rs

```diff
@@ -0,0 +1,85 @@
+// Zinc, the bare metal stack for rust.
+// Copyright 2014 Vladimir "farcaller" Pouzanov <farcaller@gmail.com>
```

Comment:
Actually, given it's just a subset from https://github.com/hackndev/zinc/blob/master/src/hal/cortex_m3/isr.rs (I just removed the reserved stuff), I don't mind leaving the current copyright notice. :smile: